Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -368,27 +387,27 @@ extension PlaceCarouselViewController: EventsProviderDelegate {
 
 extension PlaceCarouselViewController: PlacesProviderDelegate {
     func placesProviderWillStartFetchingPlaces(_ controller: PlacesProvider) {
-        // TODO placeholder for the waiting state.
-        if self.places.count == 0 {
-            headerView.numberOfPlacesLabel.text = "Waiting"
-        }
     }
 
     func placesProviderDidFinishFetchingPlaces(_ controller: PlacesProvider) {
-        // no op
         if self.places.isEmpty {
             headerView.numberOfPlacesLabel.text = "No Places Found"
```

Comment:
But if we've gone from having places to having no places, I think we need to tell the user that there are no places in their new location rather than just leaving them with a blank app.